### PR TITLE
Remove v1beta1 ingress docs

### PR DESCRIPTION
**Pull request by lamminade**
#### Changes made

- remove v1beta1 content since support was removed
- fix broken table in k8s extension api docs for services 

#### Related PRs
- [ksdk pr 1 (removes v1beta1 support bc upgrades k8s api version)](https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/265)
- [ksdk pr 2 (changes URLs)](https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/282)

### CODE REVIEW

File: source/includes/kubernetes/_k8_ingresses.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+
 <!-------------------- LIST INGRESSES V1Beta1 -------------------->
+<!-- 
 
 #### List ingresses V1Beta1
 
 ```shell
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/api/v2/services/a_service/an_environment/ingresses"
```
Comment (fyrnaga) [inline source/includes/kubernetes/_k8_ingresses.md]: i am confused. I though we kept the /ingress for beta1.. changing would mean the client would need to change all their call.

Comment (fyrnaga) [reply source/includes/kubernetes/_k8_ingresses.md]: isn't it only the view builder that is changing?

Comment (lamminade) [reply source/includes/kubernetes/_k8_ingresses.md]: whoops it was confusing but you're right
reverted

**Review by fyrnaga — APPROVED** [review]

File: source/includes/kubernetes/_k8_ingresses.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ### Ingresses V1Beta1
 
+Ingress v1beta1 has been deprecated in this API as of 26.7.0 following the removal of support by kubernetes. Please use the <a href="#kubernetes-ingresses-v1">ingress v1 api instead</a>.
+
 <!-------------------- LIST INGRESSES V1Beta1 -------------------->
+<!--
```
Comment (HassanHaidar) [inline source/includes/kubernetes/_k8_ingresses.md:6]: what is the purpose of this? are we commenting out the contents of v1beta1?

**Review by HassanHaidar — APPROVED** [review]
Looks good 👍

File: source/includes/kubernetes/_k8_ingresses.md (hunk 2 above)
Comment (lamminade) [reply source/includes/kubernetes/_k8_ingresses.md:6]: yeah if you recall we removed that code so if you call any of those APIs you will simply get this:
image

**Pull request merged**